diff --git a/kippo.tac b/kippo.tac
index 92bbdb5..98b8969 100644
--- a/kippo.tac
+++ b/kippo.tac
@@ -14,10 +14,6 @@ from twisted.application import internet, service
 from twisted.cred import portal
 from twisted.conch.ssh import factory, keys
 
-if os.name == 'posix' and os.getuid() == 0:
-    print 'ERROR: You must not run kippo as root!'
-    sys.exit(1)
-
 if not os.path.exists('kippo.cfg'):
     print 'ERROR: kippo.cfg is missing!'
     sys.exit(1)
